Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -124,6 +124,7 @@ void JitArm64::GenerateAsm()
 
   // Call C version of Dispatch().
   STR(INDEX_UNSIGNED, DISPATCHER_PC, PPC_REG, PPCSTATE_OFF(pc));
+  MOVP2R(X0, reinterpret_cast<void*>(this));
```

Comment:
This is likely fine, but I want to be sure: Do you need a reinterpret cast to the base class JitBase before?

Reply:
No, it's not necessary